Packs/Okta/Playbooks/playbook-Okta_-_User_Investigation.yml

```diff
@@ -1365,6 +1382,213 @@ tasks:
     quietmode: 2
     isoversize: false
     isautoswitchedtoquietmode: false
+  "40":
+    id: "40"
+    taskid: 25a5abbc-d30d-4012-8acf-75488d58aed8
+    type: regular
+    task:
+      id: 25a5abbc-d30d-4012-8acf-75488d58aed8
+      version: -1
+      name: Set Number Of Okta Failed Logon to Zero
+      description: Set a value in context under the key you entered.
+      scriptName: Set
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "18"
+    scriptarguments:
+      key:
+        simple: NumOfOktaFailedLogon
+      value:
+        simple: "0"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -1020,
+          "y": 805
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 2
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "41":
+    id: "41"
+    taskid: 13072b32-dac0-4c4d-8c5a-cc0b5de7d1a0
+    type: title
+    task:
+      id: 13072b32-dac0-4c4d-8c5a-cc0b5de7d1a0
+      version: -1
+      name: Failed Logon from ASN Search
+      type: title
```

Comment:
you need to addd a check that the ASN is not empty

Reply:
Done.